includes/class-newspack-popups.php

```diff
@@ -174,9 +175,15 @@ public static function display_post_states( $post_states, $post ) {
 
 			$query = new WP_Query(
 				[
-					'post_type'      => self::NEWSPACK_PLUGINS_CPT,
-					'post_status'    => 'publish',
-					'posts_per_page' => 1,
+					'post_type'        => self::NEWSPACK_PLUGINS_CPT,
+					'post_status'      => 'publish',
+					'posts_per_page'   => 1,
+					'category__not_in' => get_terms(
+						'category',
+						[
+							'fields' => 'ids',
+						]
+					),
```

Comment:
The `category__not_in` part is brutal performance-wise. This should work better IMO:
```
'tax_query' => [
    [
        'taxonomy' => 'category',
        'operator' => 'NOT EXISTS',
    ],
],
```

Reply:
When I try the `tax_query` approach I get the following warning: 

```
warning - Detected usage of tax_query, possible slow query. (WordPress.DB.SlowDBQuery.slow_db_query_tax_query)
```

Reply:
`category__not_in` is shorthand for the following tax query under the hood; it's just not caught by the linter:
```
[
    'taxonomy' => 'category',
    'operator' => 'NOT IN',
    'value' => [ 1, 2, 3, 4, 5, 6, 7, 8, 9, ... ] // Every single category on the site.
]
```

Reply:
Change made here: https://github.com/Automattic/newspack-popups/pull/15/commits/a73e69d1bee5f388e862020c0409d867a66b416a